deploy/helm/stackable-cockpit/templates/service.yaml

```diff
@@ -6,9 +6,10 @@ metadata:
   labels:
     {{- include "operator.labels" . | nindent 4 }}
 spec:
+  type: {{ .Values.service.type }}
   selector:
     {{- include "operator.selectorLabels" . | nindent 4 }}
   ports:
     - name: http
       port: 80
-      containerPort: http
+      targetPort: 8000
```

Comment:
```suggestion
      targetPort: http
```

Reply:
Sorry, I though of requesting your review as well but did not want to bother you too much :/